### DISCUSSION TITLE
Help

### DISCUSSION THREAD

**Discussion by EC-Art-Philosophy2023** [community/community#111297] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

I've been trying to do this so many times by myself. But it is time for me to get some help. For any newcomers, please do not clone or fork from my repositories. it is not a very good idea. I thank you for finding interest in my repositories, but do not use it. fork from main. my repositories will throw you off. I'm trying to develop an application with many AI containers. 
If anyone interested in building something to help middle-class working people to build a more reliable income stream and be a part of something meaningful. feel free to reach out. looking for fullstack developers, project manager, and AI data analysts. introduce yourself, the more the merrier. 
reach out to me via email, or X.com @Investforgain12

@sampottinger

**Comment by lin20260407** [comment]
push!

**Comment by ghostinhershell** [comment]
Hi @EC-Art-Philosophy2023, we’re glad you’re here 😀

Thanks for sharing your project with the community. Unfortunately, we currently do not allow self-promotion or advertising on the Community Discussions. We want to make sure there is space for users to ask questions without overwhelming them with other conversations. Thank you for helping us maintain a productive and tidy community for all our members.
